File: Builder_Pattern/Java/Director.java
public class Director {
    public Car constructSportsCar() {
        return Car.getBuilder()
                .setCarType("Sports Car")
                .setSeats(2)
                .setEngine("Powerful Engine")
                .setTransmission("Automatic")
                .setTripComputer("Advanced Trip Computer")
                .setGpsNavigator("GPS with Navigation System")
                .build();
    }

    public Car constructSUV() {
        return Car.getBuilder()
                .setCarType("SUV")
                .setSeats(5)
                .setEngine("Fuel Efficient Engine")
                .setTransmission("Manual")
                .setTripComputer("Basic Trip Computer")
                .setGpsNavigator("Standard GPS")
                .build();
    }

}

File: Builder_Pattern/Python/Director.py
from Car import Car
class Director:
    def construct_sports_car(self) -> Car :
        return Car.get_builder()\
            .set_car_type("Sports Car")\
            .set_seats(2)\
            .set_engine("Powerful Engine")\
            .set_transmission("Automatic")\
            .set_trip_computer("Advanced Trip Computer")\
            .set_gps_navigator("GPS with Navigation System")\
            .build()
    

    def construct_SUV(self) -> Car:
        return Car.get_builder()\
            .set_car_type("SUV")\
            .set_seats(5)\
            .set_engine("Fuel Efficient Engine")\
            .set_transmission("Manual")\
            .set_trip_computer("Basic Trip Computer")\
            .set_gps_navigator("Standard GPS")\
            .build()
    
